### PR TITLE
Eng-512 oteltests

### PR DESCRIPTION
**Pull request by manrodrigues**
tests are added with tag @smoke_otel_backend

set not default variables in test_config.ini:

```
backend_type=otlphttp
agent_docker_image=orbcommunity/orb-agent-full
agent_docker_tag=develop
otlp_publisher_username=12345678
otlp_publisher_key=<pass>
otlp_publisher_endpoint=https://otlp-gateway-prod-us-central-0.grafana.net/otlp
 ```

Tests behavior need to be modified according to expected results in the future

### CODE REVIEW

**Review by lpegoraro — APPROVED** [review]
LGTM

File: python-test/features/integration_config_file_otel_backend.feature [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        And 1 Agent Group(s) is created with 1 orb tag(s) (lower case)
+        And 1 policies with otel backend and yaml format are applied to the group
+    When an agent with otel backend is self-provisioned via a configuration file on port available with matching 1 group agent tags and has status online. [Overwrite default: True. Paste only file: True]
+        And otel state is running
+    Then 1 dataset(s) have validity valid and 0 have validity invalid in 30 seconds
+        And this agent's heartbeat shows that 1 groups are matching the agent
+        And the container logs should contain the message "completed RPC subscription to group" within 30 seconds
+        And this agent's heartbeat shows that 1 policies are applied and all has status running
+        And the container logs contain the message "policy applied successfully" referred to each policy within 30 seconds
+#        And the container logs that were output after all policies have been applied contain the message "scraped and published telemetry" referred to each applied policy within 180 seconds
```
Comment (mfiedorowicz) [inline python-test/features/integration_config_file_otel_backend.feature:18]: Is there any particular reason behind this step being commented out here and in the next scenario?

File: python-test/features/integration_config_file_otel_backend.feature [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        And referred sink must have active state on response within 180 seconds
+        And remove the agent .yaml generated on each scenario
+
+
+@smoke_otel_backend
+Scenario: provisioning agent with otel backend and applying 2 policy (config file - auto_provision=true)
```
Comment (mfiedorowicz) [inline python-test/features/integration_config_file_otel_backend.feature:24]: nit: s/2 policy/2 policies

File: python-test/features/integration_config_file_otel_backend.feature (hunk 1 above)
Comment (manrodrigues) [reply python-test/features/integration_config_file_otel_backend.feature:18]: reason is that log wasn't implemented yet when I wrote these scenarios and I'm not sure if they are accurate to http check, but would like to highlight that they exist for pktvisor. Do you or pegoraro already know how is the current state of this? I will fix

Comment (lpegoraro) [reply python-test/features/integration_config_file_otel_backend.feature:18]: @manrodrigues @mfiedorowicz, a thought.

We currently have 2 log entries, which do you think is best for asserting that metrics were scraped successfully?

```
{"level":"info","ts":"2023-12-01T13:47:08.558Z","caller":"otlpmqttexporter/otlp.go:171","msg":"scraped metrics for policy","policy":"github_check","policy_id":"cc7b31a2-8e57-4f1c-9504-7ef98b9f660b"}
{"level":"info","ts":"2023-12-01T13:47:08.662Z","caller":"otlpmqttexporter/otlp.go:171","msg":"scraped metrics for policy","policy":"otel_orb_std","policy_id":"8f425ce7-2ac9-4f99-b3bd-7e2c1fafd389"}
{"level":"info","ts":"2023-12-01T13:47:08.674Z","caller":"otlpmqttexporter/otlp.go:171","msg":"scraped metrics for policy","policy":"new_std","policy_id":"034d9745-d96a-42fc-904f-3fff74c5cc09"}
{"level":"info","ts":"2023-12-01T13:47:08.790Z","caller":"otlpmqttexporter/otlp.go:305","msg":"scraped and published telemetry","topic":"channels/98b27953-0404-4507-b430-11467cea2188/messages/otlp/otel/m/e","payload_size_b":1411,"compressed_payload_size_b":408}
{"level":"info","ts":"2023-12-01T13:47:08.908Z","caller":"otlpmqttexporter/otlp.go:305","msg":"scraped and published telemetry","topic":"channels/98b27953-0404-4507-b430-11467cea2188/messages/otlp/otel/m/e","payload_size_b":2470,"compressed_payload_size_b":467}
```

Comment (manrodrigues) [reply python-test/features/integration_config_file_otel_backend.feature]: i would like to have `scraped and published telemetry` by policy, to be compatible with pktvisor
on that note, what would be the reason to have `scraped metric`and then `scraped telemetry`. how are they different?

File: python-test/features/integration_config_file_otel_backend.feature (hunk 2 above)
Comment (manrodrigues) [reply python-test/features/integration_config_file_otel_backend.feature]: thanks! it's fixed (i guess)

**Review by mfiedorowicz — APPROVED** [review]

**Pull request closed without merging**